lib/Hooks/UserHooks.php

```diff
@@ -200,6 +200,8 @@ public function postDeleteUser($params) {
 		if (App::isEnabled('encryption')) {
 			$this->keyManager->deletePublicKey($params['uid']);
 		}
+
+		\OC::$server->getEncryptionKeyStorage()->deleteAltUserStorageKeys($params['uid']);
```

Comment:
Could the key storage be injected?

Reply:
Should this piece be executed only if the encryption app is enable? (just asking)